components/tooltip/README.md

```diff
@@ -77,6 +77,7 @@ In the following example to constrain the tooltip to the dashed boundary we can
 ```
 
 **Advanced Properties:**
+* `announced` (Boolean) - announce the tooltip inner text to screen reader users when the tooltip is shown. Use with custom elements.
```

Comment:
There is an `accessibility` section in the README that should also be updated to basically say use this if only if none of the other options mentioned work.